<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPage</class>
 <widget class="QMainWindow" name="LoginPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Fovea - Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/foveaicon.png</normaloff>:/images/foveaicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f9f9f9;</string>
  </property>
  <property name="iconSize">
   <size>
    <width>30</width>
    <height>30</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="6">
     <widget class="QLabel" name="loginErrorLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14pt;
color: red;</string>
      </property>
      <property name="text">
       <string>Incorrect creditionals.  Please try again.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="3">
     <widget class="QCheckBox" name="stay_signed_in">
      <property name="styleSheet">
       <string notr="true">font-size: 14pt;
color: #000000;</string>
      </property>
      <property name="text">
       <string>Stay Signed In</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="21" column="6">
     <widget class="QWidget" name="widget" native="true"/>
    </item>
    <item row="22" column="5">
     <widget class="QPushButton" name="forgot_pass">
      <property name="styleSheet">
       <string notr="true">font-size: 10pt;
color: #000000;
padding-top: 10%;
padding-bottom: 10%;
border: none;</string>
      </property>
      <property name="text">
       <string>Forgot Password?</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <widget class="QPushButton" name="register_now">
      <property name="styleSheet">
       <string notr="true">font-size: 10pt;
color: #000000;
padding-top: 10%;
padding-bottom: 10%;
border: none;</string>
      </property>
      <property name="text">
       <string>Register Here</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt;
color: #000000;</string>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="Fovea3D" name="fovea_icon">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>800</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QLabel" name="connectionErrorLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14pt;
color: red;</string>
      </property>
      <property name="text">
       <string>Please check your Internet connection.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt;
color: #000000;</string>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="6">
     <widget class="QLabel" name="forgot_label">
      <property name="styleSheet">
       <string notr="true">font-size: 14pt;
color: red;</string>
      </property>
      <property name="text">
       <string>Please enter your user name to proceed.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="App">
      <property name="styleSheet">
       <string notr="true">padding-left:10%;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt; font-weight:600; color:#000000;&quot;&gt;Sign in&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="21" column="5">
     <widget class="QPushButton" name="Sign_In">
      <property name="styleSheet">
       <string notr="true">font-size: 18pt;
color: #000000;
padding-top: 10%;
padding-bottom: 10%;</string>
      </property>
      <property name="text">
       <string>Sign In</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" colspan="2">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">padding-right: 40%;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:32pt; font-weight:600; color:#e6bb4d;&quot;&gt;Fovea.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="2" colspan="4">
     <widget class="QLineEdit" name="password_edit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="4">
     <widget class="QLineEdit" name="user_name_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Fovea3D</class>
   <extends>QOpenGLWidget</extends>
   <header>fovea3d.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>user_name_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
